### DISCUSSION TITLE
Maximum Model Top for the HRRR?

### DISCUSSION THREAD

**Discussion by Tntaviwx7** [NOAA-GSL/GSL-Regional-Model-Forum#10] [Q&A] [answered]
Hi,

I'm trying to run a WRF simulation with the highest model top (p_top_requested) setting possible when initializing with HRRR data for a high altitude modeling test case. 

Based on some documentation I've read, there is a discrepancy of what the actual model top is for the HRRR. Some sources state it is 15 hPa (https://www.weather.gov/media/notification/pdf2/scn20-46rapv5_hrrr_v4aab.pdf; https://rapidrefresh.noaa.gov/Diag-vars-NOAA-TechMemo.pdf), while other sources state it is 50 hPa (NOMADS). 

When using HRRR isobaric data from NOMADS, I ran a test case to see if I could run a simulation with a p_top_requested setting of 1500. Upon testing, real.exe failed stating that 15 hPa or p_top_requested = 1500 is not available in the input data. Upon further investigation, the highest level (physically) in these input data is 5000 Pa (50mb/50hPa) which is confirmed on NCEP's site as well as when examining the metadata. Real.exe also only ran when p_top_requested was 5000 Pa (50mb/50hPa) or greater (6000, 7000, 8000). Other sites such as AWS, Google, Utah Archive, and THREDDS point to similar conclusions for the model top height of the HRRR too.

However, I also tried running another test case, but instead of using HRRR isobaric data, I used HRRR native model levels with p_top_requested = 1500, which is equivalent to 15 hPa, and it ran successfully. 

My question is, what is the actual model top of the HRRR - is it 50 hPa or 15 hPa?

Thank you!

**Comment by EricJames-NOAA** [comment, marked as answer]
Hi, thanks for your questions!  Yes, the model top for the HRRR is at 15 hPa for HRRRv4.  However, the isobaric output only extends up to 50 hPa.  For this reason, and because there are more vertical levels and no vertical interpolation issues, we always recommend using native model level output to initialize downstream simulations.  

You can find more info on RAPv5/HRRRv4 here:
https://rapidrefresh.noaa.gov/pdf/Alexander_AMS_NWP_2020.pdf

Thanks
Eric

**Reply by Tntaviwx7** [reply]
Thank you for the explanation - it is much appreciated!
